Tests/Worker/AnnotatedWorker.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Conductor.Client.Worker;
+using conductor_csharp.test.Helper;
+
+namespace Tests.Worker
+{
+    [WorkerTask]
```

Comment:
Do we need to annotate the class itself?

Reply:
Yes, there is a condition in DiscoverWorkers() to check if the types(classes, interfaces, structs, etc.) are marked with a custom attribute WorkerTask.